src/commenttarget.js

```diff
@@ -0,0 +1,63 @@
+const SEPARATOR = '/';
+
+async function parseCommentTarget(opts = {}) {
+  const { commitSha: commit, pr, target, drv } = opts;
+
+  let commentTarget = target;
+  // Handle legacy comment target flags.
+  if (commit) {
+    drv.warn(
+      'cml: the --commitSha flag will be deprecated, please use --target="commit/<sha>"'
```

Comment:
```suggestion
      `Deprecation warning: use --target="commit${SEPARATOR}<sha>" instead of --commit-sha=<sha>`
```

or can we introduce a new proper warning class for these kinds of messages (similar to Python `DeprecationWarning`)?

Reply:
Fixed in https://github.com/iterative/cml/pull/1228/commits/dcb37b323fc75872563c163f260bdaf534e76747